### ISSUE TITLE
OcTimerLib inclusion adds noticeable delay during startup

### ISSUE THREAD

**Issue by vit9696** [acidanthera/bugtracker#526]
On some platforms like Intel before Skylake, Intel Xeon W/Scalable, and AMD we use ACPI PM timer to determine CPU frequency. This is done by OcTimerLib in its constructor to provide timer interface in our code.

As a result currently any binary referencing OcTimerLib directly or indirectly (e.g. via OcBootManagementLib) may get a ~100 ms delay during startup.

* I fixed the problem in BOOTx64.efi (https://github.com/acidanthera/OpenCorePkg/commit/b47ab162bda6dcaf5f07901d349b0ae0db82c90d) and FwRuntimeServices.efi (https://github.com/acidanthera/AppleSupportPkg/commit/e88beade3bd85a766127b889e35e0a8c0205740a) by simply removing the dependency, which is not required.
* ApfsJumpStart will be integrated into OpenCore when ready, so it should be out of scope (https://github.com/acidanthera/bugtracker/issues/385).
* OpenCore itself needs OcTimerLib and the calculation, but we may not want to calculate the ACPI PM frequency in RELEASE builds on platforms that do not need it (desktop and server Skylake+), this is still an issue.
* Other software like third-party GUI will need to reference OcBootManagementLib, and for that we want to make OcTimerLib do its job only once by installing a protocol or a volatile UEFI variable.

**Comment by Download-Fritz** [comment]
The contents of OC_CPU_INFO are implicitly related (i.e. now it may store several frequency, i.e. CPUFrequency based on ART and ACPU PM). Should it just be stripped to the minimum used by dependencies such as the SMBIOS code?
